src/App.vue

```diff
@@ -658,27 +662,27 @@ export default {
       )
 
       // Set parties
-      this.storeParties({ data: { parties: incorporationApplication.parties || [] } })
+      this.storeParties({ data: { parties: application.parties || [] } })
 
       // add this as a filing (for Filing History List)
       const filingItem: ApiFilingIF = {
         availableOnPaperOnly: header.availableOnPaperOnly,
         businessIdentifier: business.identifier,
-        commentsCount: ia.commentsCount,
-        commentsLink: ia.commentsLink,
-        displayName: this.filingTypeToName(FilingTypes.INCORPORATION_APPLICATION),
-        documentsLink: ia.documentsLink,
+        commentsCount: filedApplication.commentsCount,
+        commentsLink: filedApplication.commentsLink,
+        displayName: this.filingTypeToName(header.name),
+        documentsLink: filedApplication.documentsLink,
         effectiveDate: this.apiToUtcString(header.effectiveDate),
         filingId: header.filingId,
-        filingLink: ia.filingLink,
+        filingLink: filedApplication.filingLink,
         isFutureEffective: header.isFutureEffective,
-        name: FilingTypes.INCORPORATION_APPLICATION,
+        name: header.name,
```

Comment:
Did something complain about this? I thought the old code was legit. LMK if not.

Reply:
I don't think there was anything wrong with it, however the Sonar Code Smells was saying that it was unnecessary.

Reply:
Oh I see... `header.name` is already type FilingTypes.

It's fine. I thought SonarCloud was seeing something new. But it's me.